servers/zms/src/main/java/com/yahoo/athenz/zms/DBService.java

```diff
@@ -1230,7 +1230,54 @@ void executeDeleteMembership(ResourceContext ctx, String domainName, String role
             }
         }
     }
-    
+
+    void executeDeletePendingMembership(ResourceContext ctx, String domainName, String roleName,
+                String normalizedMember, String auditRef, String caller) {
+
+        // our exception handling code does the check for retry count
+        // and throws the exception it had received when the retry
+        // count reaches 0
+
+        for (int retryCount = defaultRetryCount; ; retryCount--) {
+
+            try (ObjectStoreConnection con = store.getConnection(true, true)) {
```

Comment:
should this be autoCommit=false?

Reply:
For single operations, we have auto commit enabled.